libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.cpp

```diff
@@ -77,7 +77,7 @@ const AP_Param::GroupInfo AC_AttitudeControl_Multi::var_info[] = {
 
     // @Param: RAT_PIT_P
     // @DisplayName: Pitch axis rate controller P gain
-    // @Description: Pitch axis rate controller P gain.  Converts the difference between desired pitch rate and actual pitch rate into a motor speed output
+    // @Description: Pitch axis rate controller P gain.  Corrects in proportion to the difference between the desired pitch rate vs actual pitch rate output
```

Comment:
Corrects in proportion to the difference between the desired pitch rate vs actual pitch rate

Reply:
fixed